chunk/chunk_store_test.go

```diff
@@ -56,6 +56,7 @@ func TestChunkStore(t *testing.T) {
 		{"v4 schema", v4Schema},
 		{"v5 schema", v5Schema},
 		{"v6 schema", v6Schema},
+		{"v7 schema", v7Schema},
```

Comment:
Yep, I've added below in `TestChunkStoreMetricNames` because the other schemas don't support these queries.

`{}` is not valid on Prometheus - at least one label or a metric name is required.